app/model/Exceptions.scala

```diff
@@ -108,6 +108,13 @@ object Exceptions {
 
   case class TooManySessionIdsException(m: String) extends Exception(m)
 
+  case class SchemeSpecificAnswerNotFound(m: String) extends Exception(m)
```

Comment:
Yeah - that text is wrapped into a 5XX/4XX response body, which should never be displayed on the frontend, but rather handled just by exception type